builtin/providers/aws/resource_aws_s3_bucket.go

```diff
@@ -538,15 +538,18 @@ func resourceAwsS3BucketRead(d *schema.ResourceData, meta interface{}) error {
 		}
 	}
 
-	//read the acceleration status
-	accelerate, err := s3conn.GetBucketAccelerateConfiguration(&s3.GetBucketAccelerateConfigurationInput{
-		Bucket: aws.String(d.Id()),
-	})
-	log.Printf("[DEBUG] S3 bucket: %s, read Acceleration: %v", d.Id(), accelerate)
-	if err != nil {
-		return err
+	s3HostedRegion := meta.(*AWSClient).region
+	if s3HostedRegion != "cn-north-1" && s3HostedRegion != "us-gov-west-1" {
```

Comment:
Hard-coding regions means that once AWS actually starts supporting this feature in `cn-north-1` & `us-gov-west-1`, we'll have to patch it again and users will have to wait for the next release :/

How about silently ignoring `UnsupportedArgument` errors instead? i.e. we'd still error out if _any other_ error comes from AWS API, but this error code seems to be unique enough that it shouldn't have any unintended consequences (like hiding actual errors).